app/sider/SendEndring.tsx

```diff
@@ -53,7 +53,7 @@ export default function SendEndringSide() {
       settEndringsmeldingMottattDato(actionData.data.mottattDato);
       navigate(hentPathForSteg(ytelse, ESteg.KVITTERING));
     } else {
-      settErResponseOK(false);
+      settFeilKode(actionData.feilKode || null);
     }
```

Comment:
Hvorfor dette? 😅
Er det fordi actionData.feilkode ikkje kan returnere null av default?
Forstår ikkje heile bruken av || her. Blir ikkje dette en boolean?

Reply:
Bra du spurte her! Tanken var at det skulle være en valideringsfeil eller null for alle andre feil typ i starten men da blir det jo vanskelig å bestemme når alerten skal vises eller ikke. Nå som det blir satt en feilkode uansett i sendEndringAction.server.ts skal ikke denne kunne bli null. Det er da enten MANGLER_TEKST... eller INGEN_VALIDERINGSFEIL.

Reply:
Okay, den må være der ettersom IPostResponse enten er status og data, eller status og feilkode så kommer det feilkode om at actionData.feilKode kan være undefined. Allikevel vet jo vi at om status ikke er ok, så kommer det et response med status og feilkode (der INGEN_VALIDERINGSFEIL, er alle andre feil enn de spesifike vi har gitt tilbake) så den vil da sette en feilkode.

Reply:
Korleis veit man at verdien blir X og at det hele ikkje blir boolean? Har det noe med typen til X?